code/modules/mob/living/carbon/carbon.dm

```diff
@@ -66,19 +66,36 @@
 			take_bodypart_damage(5 + 5 * extra_speed, check_armor = TRUE, wound_bonus = extra_speed * 5)
 		else if(!iscarbon(hit_atom) && extra_speed)
 			take_bodypart_damage(5 * extra_speed, check_armor = TRUE, wound_bonus = extra_speed * 5)
+		playsound(src,'sound/weapons/punch1.ogg',50,TRUE)
```

Comment:
Why's this here? Looks like it will cause two punch sound effects to play

Reply:
I believe at present there are some dense objects it doesn't make any noise at all if you hit jt. Lemme fiddle with it.